src/main/controller/ExerciseController.ts

```diff
@@ -0,0 +1,33 @@
+import { Request, Response } from 'express';
+import db from '../models';
+import { Route } from '../constant/Route';
+
+export class ExerciseController {
+  // (4 - EXP + WP) Retrieve detailed information about an exercise (name, video link, desc, tags)
+  getExerciseDetailed = async (req: Request, res: Response): Promise<any> => {
+    try {
+      if (!req.params.exerciseID) {
+        throw `Missing exerciseID param. usage: /${Route.EXERCISES}/:exerciseID(\\d)`;
+      }
+
+      // TODO: verify return
```

Comment:
Do you mind clarifying what verify return means?

Reply:
The verify return is just a reminder that we still need to include `orderNum`, and that my algorithm can send the exercises in the correct order.

Reply:
Are you planning to add orderNum in this PR?

Reply:
I think we can leave it out for now, considering we don't have an `orderNum` column in our database yet

Reply:
I'm leaning towards doing it here cause it should be faster to do it now vs later- should be quick and easy though! We just need to add a new migration and edit the model/entity files

Reply:
Sure! I'm not entirely sure how to add the new migration and edit the model files though.

Reply:
We're essentially just adding another column via a migration & updating the model files- this might help! You can look at this for migrations - https://sequelize.org/v3/docs/migrations/